source/stylesheets/refills/unstyled/_sliding-panel.scss

```diff
@@ -0,0 +1,36 @@
+.sliding-panel-content {  
+  $sliding-panel-width: 220px;
+
+  @include position(fixed, 0px auto 0px 0px);
+  @include size($sliding-panel-width 100%);
+  @include transform(translateX(- $sliding-panel-width));
+  @include transition(all 0.25s linear);
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.